bin/migrate.sh

```diff
@@ -155,37 +161,15 @@ echo Updating known shared imports...
 print-files "$DST_DIR" | xargs sed -i -e "s|import {cliVersion} from '../../version'|import {cliVersion} from '@datadog/datadog-ci/src/version'|g"
 echo Done
 
-echo 6. Add plugin folder to tsconfig.json and packages/datadog-ci/tsconfig.json
-sed -i -e 's|in the future.|in the future.\n    {\n      "path": "./'"${PLUGIN_DIR}"'"\n    },|g' tsconfig.json
-sed -i -e 's|Add Plugins Here:|Add Plugins Here:\n    {\n      "path": "../'"plugin-$SCOPE"'"\n    },|g' packages/datadog-ci/tsconfig.json
-echo Done
-
-echo 7. Update CI configuration
-
-CI_FILE=".github/workflows/ci.yml"
-BASE_LINE='"@datadog/datadog-ci-base": "file:./artifacts/@datadog-datadog-ci-base-${{ matrix.version }}.tgz",'
-PLUGIN_LINE="              \"$PLUGIN_PKG\": \"file:./artifacts/@datadog-datadog-ci-plugin-$SCOPE-\${{ matrix.version }}.tgz\","
-
-if grep -q "$BASE_LINE" "$CI_FILE"; then
-  # Insert the plugin package line after the datadog-ci-base line in ci.yml
-  sed -i -e "s|$BASE_LINE|$BASE_LINE\\n$PLUGIN_LINE|" "$CI_FILE"
-  echo "Updated .github/workflows/ci.yml to include $PLUGIN_PKG"
-else
-  echo "Could not find base line in .github/workflows/ci.yml -- please add the following line manually:
-$PLUGIN_LINE" 
-fi
-
-echo 8. Update CODEOWNERS
+echo 6. Update CODEOWNERS
```

Comment:
This script just updates the CODEOWNERS file, then `yarn lint:packages --fix` formats it properly.